modules/index/build.gradle

```diff
@@ -81,6 +81,7 @@ dependencies {
     integrationTestImplementation testFixtures(project(':ignite-table'))
     integrationTestImplementation testFixtures(project(':ignite-storage-api'))
     integrationTestImplementation testFixtures(project(':ignite-catalog'))
+    integrationTestImplementation testFixtures(project(':ignite-replicator'))
```

Comment:
Why do we need this dependency?

Reply:
The new `ReplicaTestUtils` is placed there